microsoftsccm.json

```diff
@@ -23,6 +23,10 @@
     "app_wizard_version": "1.0.0",
     "pip_dependencies": {
```

Comment:
Why don't you just get rid of this one? Does the script always restore it?

Reply:
If I make any changes for dependencies and rebuild them, it'll be created all over again. I think we should tweak precommit hooks instead of manually removing deprecated keys.